rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -606,6 +606,13 @@ namespace vk
 			enabled_features.logicOp = VK_FALSE;
 		}
 
+		if (!pgpu->features.textureCompressionBC)
```

Comment:
To avoid crashes we should check the vendor.

Reply:
Done. I did not add it on VKGSRender.cpp:2408 on s_allowed_vendors for "Preserving an active renderpass across a transfer operation" since it's untested and the driver is too unstable as is

Reply:
It should be a TBDR as well like other mobile GPUs, those ones we can never preserve the renderpass.